packages/dcos-integration-test/extra/test_dcos_diagnostics.py

```diff
@@ -401,51 +396,6 @@ def test_dcos_diagnostics_report(dcos_api_session):
         assert len(report_response['Nodes']) > 0
 
 
-def _get_bundle_list(dcos_api_session):
```

Comment:
Can you explain the motivation for moving these help functions to `test_helpers.py`? They don't seem to be useful to tests in other files. And we'd want to clean them up a bit before we publish them as general-purpose test utils (they need docstrings, `wait_for_diagnostics_job()` shouldn't be returning info by mutating arguments, `check_json()` shouldn't call `raise_for_status()`, etc.)

Reply:
I agree. The main reason I moved this functions to helper is to solve Python issues with sources path. The test I'm adding here is moved from dcos-enterprise but IMO both OSS and EE version should contain this test. But if I leave this helpers untouched then this tests in EE version fails due to dependencies (`ModuleNotFound`) because OSS tests are imported in sub-directory in EE. 

I agree, this functions are not util functions and probably should not be available in that file. On the other hand fixing it this way is better than something like this:
```python
import sys
sys.path.insert(0,'.')
```

Reply:
IIRC these functions are imported from dcos-enterprise, hence a move to test_helpers makes sense.